en/advanced/gimbal_control.md

```diff
@@ -1,82 +1,64 @@
 For this configuration you would set:
 - [MAV_1_CONFIG](../advanced_config/parameter_reference.md#MAV_1_CONFIG) to **TELEM2** (if `MAV_1_CONFIG` is already used for a companion computer (say), use `MAV_2_CONFIG`).
 - [MAV_1_MODE](../advanced_config/parameter_reference.md#MAV_1_MODE) to **NORMAL**
 - [SER_TEL2_BAUD](../advanced_config/parameter_reference.md#SER_TEL2_BAUD) to manufacturer recommended baud rate.
 
-This will enable the user to command the gimbal using [MAV_CMD_DO_MOUNT_CONTROL](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_MOUNT_CONTROL) and [MAV_CMD_DO_MOUNT_CONFIGURE](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_MOUNT_CONFIGURE).
-
+This will enable MAVLink to command the gimbal using [MAV_CMD_DO_MOUNT_CONTROL](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_MOUNT_CONTROL) and [MAV_CMD_DO_MOUNT_CONFIGURE](https://mavlink.io/en/messages/common.html#MAV_CMD_DO_MOUNT_CONFIGURE).
 
-## Gimbal on Flight Controller (MNT_MODE_OUT=AUX)
 
-The gimbal can be connected to the Flight controller AUX ports by setting the output mode to `MNT_MODE_OUT=AUX`.
+## Gimbal on Flight Controller PWM Output (MNT_MODE_OUT=AUX)
 
-A mixer file is required to define the mapping for the output pins and the [mount mixer](https://github.com/PX4/PX4-Autopilot/blob/main/ROMFS/px4fmu_common/mixers/mount.aux.mix) is automatically selected (this overrides any AUX mixer provided by the airframe configuration).
+The gimbal can be connected to the Flight controller PWM ports by setting the output mode to `MNT_MODE_OUT=AUX`.
 
-The output assignment is as following:
-- **AUX1**: Pitch
-- **AUX2**: Roll
-- **AUX3**: Yaw
-- **AUX4**: Shutter/retract
+The output pins that are used to control the gimbal are set in the [Acuator Configuration > Outputs](/en/config/actuators.md#actuator-outputs) by selecting any three unused Actuator Outputs and assigning the following output functions:
+- `Gimbal Roll`: Output controls gimbal roll.
+- `Gimbal Pitch`: Output controls Gimbal pitch.
+- `Gimbal Yaw`: Output controls Gimbal pitch.
 
-### Customizing the mixer configuration
+For example, you might have the following settings to assign the gimbal roll, pitch and yaw to AUX1-3 outputs.
 
-:::tip
-Read [Mixing and Actuators](../concept/mixing.md) for an explanation of how mixers work and the format of the mixer file.
-:::
-
-The outputs can be customized by [creating a mixer file](../concept/system_startup.md#starting-a-custom-mixer) on the SD card named `etc/mixers/mount.aux.mix`.
-
-A basic mixer configuration for a mount is shown below.
-
-```
-# roll
-M: 1
-O:      10000  10000      0 -10000  10000
-S: 2 0  10000  10000      0 -10000  10000
-
-# pitch
-M: 1
-O:      10000  10000      0 -10000  10000
-S: 2 1  10000  10000      0 -10000  10000
-
-# yaw
-M: 1
-O:      10000  10000      0 -10000  10000
-S: 2 2  10000  10000      0 -10000  10000
-```
+![Gimbal Actuator config](../../assets/config/actuators/qgc_actuators_gimbal.png)
```

Comment:
The image is just a control actuator image at top of this page. 

How do we choose the ranges to set here for the min, max, disarmed? 

I.e. is the same as in http://docs.px4.io/main/en/config/actuators.html#output-assignment-manual - you basically wiggle the actuator test bar to confirm the gimbal moves in the right axis, then set the range such that middle is disarmed position, and there is an up range and down range around the central point. So you increase to top value that the gimbal can physically move and that is the PWM value to set, ditto for the bottom of the range?

Reply:
Yes either by testing or the gimbal docs might specify the range.

Reply:
THanks. I added a note to this effect.